hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/client/ScmClient.java

```diff
@@ -360,6 +360,12 @@ StartContainerBalancerResponseProto startContainerBalancer(
    */
   List<String> getScmRatisRoles() throws IOException;
 
+  /**
+   * Renew the expired deleted block retry count.
+   * @throws IOException
+   */
+  int renewDeletedBlockRetryCount(List<Long> txIDs) throws IOException;
```

Comment:
Since the operation is called `resetCount` in `DeletedBlockLog`, would it make sense to use `reset` instead of `renew` consistently?

Reply:
Usually commands are in verb form: e.g. `... key put`, `... volume create`, instead of `... key putter`, `... volume creator`.

So I think `ozone admin scm reset-deleted-block-retry-count` would be better.